src/features/pastelDB/__tests__/pastelDBLib.test.ts

```diff
@@ -1354,4 +1354,50 @@ describe('managePastelDatabase', () => {
       )
     }
   })
+
+  test('getTransactionsDataFromDBByPeriod should works correctly', async () => {
```

Comment:
I think  we  need to create 2 test cases  for  this  test
- Correct case  
- Error case